docs/topic-guides/cluster-configuration/elasticsearch-logging-backend.txt

```diff
@@ -0,0 +1,56 @@
+guideline on when to migrate to Elastic, and some tips for how to tune
+Elasticsearch to work best with Determined.
+
+`Elasticsearch <https://www.elastic.co/what-is/elasticsearch>`__ is a
+search engine commonly used for storing application logs for search and
+analytics. Determined supports using Elasticsearch as the storage
+backend for trial logs. Configuring Determined to use Elasticsearch is
+simple, however managing an Elasticsearch cluster as scale is an
+involved task, so this guide is recommended for users who have hit the
+limitations of the default logging backend.
+
+Using the default logging backend, with a the standard deployment using
+``det-deploy``, the cluster can ingest logs about as fast as Postgres
+can persist them. For example, with ``det-deploy aws`` using Aurora
+Serverless with 2 capacity units, ingestion speed maxes out around 10-15
+MB/s (where the database's CPU hits ~90%). To get a little more mileage
+from the default, we recommend increasing the capacity of the database.
+At a certain point, the master instance itself will become the
+bottleneck, since it has limited incoming network bandwidth for HTTP
+requests delivering logs and limited resources to process them. For
+example, ``m5.large`` Amazon instances the max bandwidth is 0.74 Gb/s.
+At this point, the master instance size can also be increased but,
+eventually, once the limits of vertical scaling are reached (likely
+around hundreds of megabytes of logs per second), it is recommended to
+move to Elasticsearch.
+
+Determined offers some additional recommendations for the Elasticsearch
+cluster configuration based on how the cluster will be used:
+
+-  Tune the default shards per index to your expected throughput (or use
+   `index templates
+   <https://www.elastic.co/guide/en/elasticsearch/reference/7.10/index-templates.html>`__).
+   Determined ships logs in Logstash format rolling over to a new index
+   each day. Depending on your log volume, the default number of shards
+   could be too many or too few shards. The general rule of thumb is not
+   to exceed 50 GB per shard while minimizing the number of shards per
+   index. For high utilization clusters, this may entail increasing the
+   shards per index and rotating indices older than a few months out of
+   the cluster periodically, to avoid the overhead accumulated from
+   having too many shards. A more in depth guide can be found `here
+   <https://www.elastic.co/guide/en/elasticsearch/reference/current/size-your-shards.html>`__.
+
+-  Though it may increase latency for end users, increasing the `refresh
+   interval
+   <https://www.elastic.co/guide/en/elasticsearch/reference/master/tune-for-indexing-speed.html#_unset_or_increase_the_refresh_interval>`__
+   may help increase total throughput.
+
+The configuration settings to enable Elastic as the trial log backend
+are described in the :ref:`cluster-configuration` reference.
```

Comment:
```suggestion
are described at :ref:`cluster-configuration`.
```
or
```suggestion
are described in the :ref:`cluster configuration <cluster-configuration>` reference.
```
(The idea is to think of page titles as opaque and either specify text at the link site or treat them like locations without requiring them to grammatically fit with the surrounding text. That way, page titles can be changed freely without causing any weird-looking sentences.)